operate/pom.xml

```diff
@@ -31,6 +31,13 @@
     <module>../distro</module>
   </modules>
 
+  <scm>
+    <connection>scm:git:https://${env.GITHUB_TOKEN_USR}:${env.GITHUB_TOKEN_PSW}@github.com/camunda/zeebe.git</connection>
```

Comment:
@cmur2 I have a doubt if we should use `${env.GITHUB_TOKEN_USR}:${env.GITHUB_TOKEN_PSW}` or `${env.GITHUB_USERNAME}:${env.GITHUB_TOKEN}`

Reply:
I'm not sure to be honest. The only references to `GITHUB_TOKEN_USR` in this repository is the `pom.xml` file itself which relies on them but there is no place that defines them, e.g. in CI: https://github.com/search?q=repo%3Acamunda%2Fzeebe+GITHUB_TOKEN_USR&type=code

I also don't see where `GITHUB_USERNAME` would come from, it is also not defined in this repository and is none of Githubs default variables.

Some months ago I migrated another Zeebe repository from Jenkins to Github actions and found `GITHUB_ACTOR` and `GITHUB_TOKEN` to be working: https://github.com/zeebe-io/flaky-test-extractor-maven-plugin/blob/main/pom.xml#L20-L25

Reply:
The `GITHUB_TOKEN_USR` variable was used in conjunction with Jenkins (when Zeebe still used Jenkins) according to https://github.com/camunda/zeebe/commit/0a63a7e4336a114a87ca356b0bfe73f06a9f9eae and is most likely not meaningful today. This makes me think that it doesn't really matter which value we put there since Zeebe seems to release fine since 3 years 😁